src/components/ScrollToTopButton/style.css

```diff
@@ -0,0 +1,18 @@
+.to-top-button {
+  z-index: 10000;
+  position: fixed !important;
+  bottom: 1rem;
+  right: 1rem;
+  background-color: white !important;
+  box-shadow: 0 0 6px rgba(0, 0, 0, 0.3);
```

Comment:
The spread of the shadow is the weird cause of colour I think

Reply:
![image](https://user-images.githubusercontent.com/60546840/105480368-85691300-5c5a-11eb-921b-32808911f03d.png)